meilisearch-release-changelog/v1.14.0.md

```diff
@@ -0,0 +1,137 @@
+                "response": [
+                    "{{embedding}}",
+                    "{{..}}"
+                ]
+            }
+        }
+    }
+}
+```
+3. Send documents to the index. They will be indexed remotely using the Hugging Face inference endpoint.
+4. Perform semantic search queries, they will be embedded using the local model fetched from the Hugging Face Hub.
+
+For more details about this feature, please refer to its [public usage page](https://www.notion.so/meilisearch/Composite-embedder-usage-14a4b06b651f81859dc3df21e8cd02a0?pvs=4#19f4b06b651f809ab3ecc0daae7d5856)
+<TBD: Find a better title>
+
+Done by @dureuill in [#5371](https://github.com/meilisearch/meilisearch/pull/5371).
+
+## Other improvements
+
+* Get multiple documents by ids by @dureuill in [#5384](https://github.com/meilisearch/meilisearch/pull/5384)
+* Support merging update and replacement operations by @Kerollmops in [#5293](https://github.com/meilisearch/meilisearch/pull/5293)
+* Display the internal indexing steps with timings on the `/batches` route by @Kerollmops in [#5356](https://github.com/meilisearch/meilisearch/pull/5356)
+* exhaustive facet search by @ManyTheFish in [#5369](https://github.com/meilisearch/meilisearch/pull/5369)
+* Reduce RAM consumption of arroy by @irevoire in https://github.com/meilisearch/arroy/pull/105
+* Cache Embeddings in Search with an experimental feature by @dureuill in [#5418](https://github.com/meilisearch/meilisearch/pull/5418)
+* Extend the batch progress view to the indexing of vectors by @irevoire in [5420](https://github.com/meilisearch/meilisearch/pull/5420)
+
+# Fixes 🐞
+
+* Support fetching the pooling method from the model configuration by @dureuill in [#5355](https://github.com/meilisearch/meilisearch/pull/5355)
+* Make sure to delete useless prefixes by @Kerollmops in [#5413](https://github.com/meilisearch/meilisearch/pull/5413)
+
+# Misc
+
+* Dependencies updates
+  * Bump Ubuntu in the CI from 20.04 to 22.04 by @Kerollmops in [#5338](https://github.com/meilisearch/meilisearch/pull/5338)
+  * Bump heed to v0.22 by @irevoire and @Kerollmops in [#5406](https://github.com/meilisearch/meilisearch/pull/5406)
+* CIs and tests
+  * Improve test performance of get_index.rs  by @DerTimonius in [#5210](https://github.com/meilisearch/meilisearch/pull/5210)
+  * Ollama Integration Tests by @Kerollmops in [#5308](https://github.com/meilisearch/meilisearch/pull/5308)
+  * Ensure the settings routes are now configurated when a new field is added to the Settings struct  by @MichaScant in [#5149](https://github.com/meilisearch/meilisearch/pull/5149)
+  * Skip a snapshot test on Windows by @Kerollmops in [#5383](https://github.com/meilisearch/meilisearch/pull/5383)
+* Misc
+  * Rename `callTrace` into `progressTrace` by @Kerollmops in [#5364](https://github.com/meilisearch/meilisearch/pull/5364)
+  * Make composite embedders an experimental feature by @dureuill in [#5401](https://github.com/meilisearch/meilisearch/pull/5401)
```

Comment:
I don't think these need their own entries, as they were done for v1.14. I'd add `#5364` (resp. `#5401`) to `#5356` (resp. `#5371`)

```suggestion
```

Reply:
Yes, you're absolutely right!